server/auth/auth_routes.py

```diff
@@ -349,9 +348,13 @@ def tally_entry_passphrase(passphrase: str):
     jurisdiction = Jurisdiction.query.filter_by(
         tally_entry_passphrase=passphrase
     ).one_or_none()
-    # TODO redirect to a nice error screen that explains they probably made a typo
     if jurisdiction is None:
-        raise NotFound()
+        return redirect(
+            "/tally-entry?"
+            + urlencode(
+                {"error": "login_link_not_found", "message": "Login link not found."}
```

Comment:
Do we use the `message` param anywhere? Maybe an artifact of an intermediate iteration of this code?